### ISSUE TITLE
No attachment records in database

### ISSUE THREAD

**Issue by inpos** [texxasrulez/Caldav_Calendar#8]
Attachment successfully uploaded and saved to server.
After save and reopen event there is uploaded attachment.
But after reload page and open event there is no any attachment.
When query select from "caldav_attachments" table in database there is no any records.
```
roundcube=# select * from caldav_attachments;
 attachment_id | event_id | filename | mimetype | size | data 
---------------+----------+----------+----------+------+------
(0 lines)

roundcube=# 

```

**Comment by texxasrulez** [comment]
Which DB are you using?

**Comment by texxasrulez** [comment]
I can confirm this on MySQL. Since first attachment issue, I have used same file for testing. If you upload file a second time, it will stay in database. That is why I couldn't recreate this until I started with a different file to attach ... 
Now I get "file_get_contents(): Filename cannot be empty" . Filenames are obviously not empty, so I will figure this out ... 

This issue is accepted and I am working on it. Thank you for your contribution ...
